RFC/src/RFC-0201_TariScript.md

```diff
@@ -484,145 +544,178 @@ Alice owns \\( C_a \\) and provides the required script to spend the UTXO as was
 
-Because Bob's wallet already knowns \\( k_Sb \\), he now knows all the values required to spend the commitment \\( C_b \\)
+Because Bob's wallet already knowns the _script private key_ \\( k_{Sb} \\), he now knows all the values required to 
+spend the commitment \\( C_b \\)
 
 For Bob's part, when he discovers one-sided payments to himself, he should spend them to new outputs using a traditional
 transaction to thwart any potential horizon attacks in the future.
 
 To summarise, the information required for one-sided transactions is as follows:
 
-| Transaction input | Symbols                               | Knowledge                                                       |
-|:------------------|:--------------------------------------|:----------------------------------------------------------------|
-| commitment        | \\( C_a = k_a \cdot G + v \cdot H \\) | Alice knows spend key and value                                 |
-| features          | \\( F_a \\)                           | Public                                                          |
-| script            | \\( \alpha_a \\)                      | Public, can verify that \\( \hash{\alpha_a} = \scripthash_a \\) |
-| script input      | \\( \input_a \\)                      | Public                                                          |
-| height            | \\( h_a \\)                           | Public                                                          |
-| script signature  | \\( s_{Sa}, R_{Sa} \\)                | Alice knows \\( k_{Sa},\\, r_{Sa} \\)                           |
-| offset public key | \\( K_{Oa} \\)                        | Not used in this transaction                                    |
+| Transaction input | Symbols                               | Knowledge                                                                                     |
+|-------------------|---------------------------------------|-----------------------------------------------------------------------------------------------|
+| commitment        | \\( C_a = k_a \cdot G + v \cdot H \\) | Alice knows the blinding factor and value                                                     |
+| features          | \\( F_a \\)                           | Public                                                                                        |
+| script            | \\( \alpha_a \\)                      | Public                                                                                        |
+| script input      | \\( \input_a \\)                      | Public                                                                                        |
+| height            | \\( h_a \\)                           | Public                                                                                        |
+| script signature  | \\( s_{S1a},s_{S2a}, R_{Sa} \\)       | Alice knows \\( k_{Sa},\\, r_{Sa} \\) and \\( k_{a},\\, v_{a} \\) of the commitment \\(C_a\\) |
```

Comment:
```suggestion
| script signature  | \\( a_{Sa},b_{Sa}, R_{Sa} \\)       | Alice knows \\( k_{Sa},\\, r_{Sa} \\) and \\( k_{a},\\, v_{a} \\) of the commitment \\(C_a\\) |
```

Reply:
I would add the contents of the challenge here

Reply:
The challenge contents are all public information, these tables are for private information.